etc/cache.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Cache/etc/cache.xsd">
```

Comment:
This file is needed to show this custom cache option on Admin panel. When it is visible on admin and enabled a merchant/admin with enough privilege can flush or clean it from there.